tutorials/plot_configuration.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+========================================
+Background information on configurations
+========================================
+
+This tutorial gives a short introduction to MNE configurations.
+"""
+import os.path as op
+
+import mne
+from mne.datasets.sample import data_path
+
+fname = op.join(data_path(), 'MEG', 'sample', 'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(fname)
+raw.crop(10)
+
+###############################################################################
+# MNE python stores configurations to a folder called `.mne` in the user's
+# home directory, These configurations include information like sample data
```

Comment:
"... home directory on Linux or AppData directory on Windows..."

You should probably cross-link to [get_config_path] (https://github.com/mne-tools/mne-python/blob/master/mne/utils.py#L1371) and/or show it by example